localization/ndt_scan_matcher/src/util_func.cpp

```diff
@@ -205,14 +205,38 @@ void popOldPose(
   }
 }
 
-Eigen::Affine3d fromRosPoseToEigen(const geometry_msgs::msg::Pose & ros_pose)
+Eigen::Affine3d from_ros_pose_to_eigen_affine3d(const geometry_msgs::msg::Pose & ros_pose)
 {
   Eigen::Affine3d eigen_pose;
   tf2::fromMsg(ros_pose, eigen_pose);
   return eigen_pose;
 }
 
-std::vector<geometry_msgs::msg::Pose> createRandomPoseArray(
+Eigen::Matrix4f from_ros_pose_to_eigen_matrix4f(const geometry_msgs::msg::Pose & ros_pose)
```

Comment:
I think it is sufficient to write just `ros_pose_to_eigen_matrix4f`

Reply:
resolved https://github.com/autowarefoundation/autoware.universe/pull/1757/commits/11c1edaaf6b7efbbf93f811ef04a944269a6bf51

Reply:
Thank you for the update

If you want an even shorter name, I think `pose_to_matrix4f` is also sufficient because we can infer from argument/return types.

Reply:
You're right, thanks. I updated the names https://github.com/autowarefoundation/autoware.universe/pull/1757/commits/75632bf55892ba2761039d23fce2a7cbe538339b.